analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/JavaDataLoaderFunctionalTest.java

```diff
@@ -55,14 +60,14 @@ public void ensureAlignmentFileCreated() throws IOException, URISyntaxException,
                 projectRoot, projectRoot.getName(),
                 Collections.singletonMap("dependencyOverride.com.yammer.metrics:*@org.acme.gradle:root", ""));
 
-        assertTrue(new File(projectRoot, AlignmentTask.GME).exists());
-        assertTrue(new File(projectRoot, AlignmentTask.GRADLE + '/' + AlignmentTask.GME_REPOS).exists());
-        assertTrue(new File(projectRoot, AlignmentTask.GME_PLUGINCONFIGS).exists());
-        assertEquals(AlignmentTask.INJECT_GME_START, TestUtils.getLine(projectRoot));
-        assertEquals(AlignmentTask.INJECT_GME_END,
-                org.jboss.gm.common.utils.FileUtils.getLastLine(new File(projectRoot, Project.DEFAULT_BUILD_FILE)));
-        assertTrue(systemOutRule.getLog()
-                .contains("Passing 1 GAVs following into the REST client api [com.graphql-java:java-dataloader:"));
```

Comment:
I've changed this log message slightly to differentiate it from the message for the other endpoint. Also, "following" seemed to be a grammatical error.

Reply:
Good idea. I changed the log message slightly in PME as well.